src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -0,0 +1,213 @@
+  keyExtractor: recordKeyExtractor,
+  dataState: new Map(),
+  searchTerm: '',
+  filterDataKeys: ['item.name'],
+  sortBy: 'itemName',
+  isAscending: true,
+  modalKey: '',
+  modalValue: null,
+});
+
+const supplierInvoiceInitialiser = pageObject => {
+  const backingData = pageObject.getTransactionBatches(UIDatabase);
+  return {
+    pageObject,
+    backingData,
+    data: backingData.sorted('itemName').slice(),
+    keyExtractor: recordKeyExtractor,
+    dataState: new Map(),
+    searchTerm: '',
+    filterDataKeys: ['itemName'],
+    sortBy: 'itemName',
+    isAscending: true,
+    modalKey: '',
+    modalValue: null,
+    hasSelection: false,
+  };
+};
+
+const supplierInvoicesInitialiser = () => {
+  const backingData = UIDatabase.objects('SupplierInvoice');
+  return {
+    backingData,
+    data: newSortDataBy(backingData.slice(), 'serialNumber', false),
+    keyExtractor: recordKeyExtractor,
+    dataState: new Map(),
+    searchTerm: '',
+    filterDataKeys: ['otherParty.name'],
+    sortBy: 'serialNumber',
+    isAscending: false,
+    modalKey: '',
+    hasSelection: false,
+  };
+};
+
+const supplierRequisitionInitialiser = requisition => {
+  const { program, items: backingData } = requisition;
+  const showAll = !program;
+
+  return {
+    pageObject: requisition,
+    backingData,
+    data: showAll
+      ? backingData.sorted('item.name').slice()
+      : backingData.filter(item => item.isLessThanThresholdMOS),
+    keyExtractor: recordKeyExtractor,
+    dataState: new Map(),
+    currentFocusedRowKey: null,
+    searchTerm: '',
+    filterDataKeys: ['item.name'],
+    sortBy: 'itemName',
+    isAscending: true,
+    modalKey: '',
+    hasSelection: false,
+    showAll,
+    modalValue: null,
+  };
+};
+
+const supplierRequisitionsInitialiser = () => {
+  const backingData = UIDatabase.objects('RequestRequisition');
+  const data = newSortDataBy(backingData.slice(), 'serialNumber');
+  return {
+    backingData,
+    data,
+    keyExtractor: recordKeyExtractor,
+    dataState: new Map(),
+    searchTerm: '',
+    filterDataKeys: ['serialNumber', 'otherStoreName.name'],
+    sortBy: 'serialNumber',
+    isAscending: true,
+    modalKey: '',
+    hasSelection: false,
+  };
+};
+
+const getPageInitialiser = page => {
+  const pageToInitialiser = {
+    customerInvoice: customerInvoiceInitialiser,
+    customerInvoices: customerInvoicesInitialiser,
+    customerRequisition: customerRequisitionInitialiser,
+    customerRequisitions: customerRequisitionsInitialiser,
+    stocktakeEditor: stocktakeEditorInitialiser,
+    stocktakeManager: stocktakeManagerInitialiser,
+    stocktakes: stocktakesInitialiser,
+    supplierInvoice: supplierInvoiceInitialiser,
+    supplierInvoices: supplierInvoicesInitialiser,
+    supplierRequisition: supplierRequisitionInitialiser,
+    supplierRequisitions: supplierRequisitionsInitialiser,
+  };
+
+  return pageToInitialiser[page];
+};
+
+export default getPageInitialiser;
```

Comment:
https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

Reply:
Yeah, I'm not a big fan of `default`. I think it's best to address this in a separate issue, for consistencies sake with other methods (e.g. `getColumns`, `getPageInfoColumns` also `default export`, so my though would be to keep `getPageInitialiser` consistent). 

If we want to go with this pattern, will also need to update the eslint config as we currently use air-bnb's `prefer-default-export`: https://github.com/airbnb/javascript#modules--prefer-default-export.

Reply:
you're right! Was going to do it here.. oopsies https://github.com/openmsupply/mobile/issues/845#issuecomment-524062278 (more just bringing this to attention than requiring some sort of change right now :))